google-cloud-bigquery/lib/google/cloud/bigquery/extract_job.rb

```diff
@@ -155,6 +155,122 @@ def destinations_file_counts
         def destinations_counts
           Hash[destinations.zip destinations_file_counts]
         end
+
+        ##
+        # Yielded to a block to accumulate changes for an API request.
+        class Updater < ExtractJob
+          ##
+          # @private Create an Updater object.
+          def initialize gapi
+            @gapi = gapi
+          end
+
+          ##
+          # @private Create an Updater from an options hash.
+          #
+          # @return [Google::Cloud::Bigquery::ExtractJob::Updater] A job
+          #   configuration object for setting query options.
+          def self.from_options table, storage_files, options = {}
+            storage_urls = Array(storage_files).map do |url|
+              url.respond_to?(:to_gs_url) ? url.to_gs_url : url
+            end
+            dest_format = options[:format]
+            if dest_format.nil?
+              dest_format = Convert.derive_source_format storage_urls.first
+            end
+            req = Google::Apis::BigqueryV2::Job.new(
+              configuration: Google::Apis::BigqueryV2::JobConfiguration.new(
+                extract: Google::Apis::BigqueryV2::JobConfigurationExtract.new(
+                  destination_uris: Array(storage_urls),
+                  source_table: table
+                ),
+                dry_run: options[:dryrun]
+              )
+            )
+
+            updater = ExtractJob::Updater.new req
+            updater.compression = options[:compression]
+            updater.delimiter = options[:delimiter]
+            updater.format = dest_format
+            updater.header = options[:header]
+            updater.labels = options[:labels] if options[:labels]
+            updater
+          end
+
+          # Sets the compression type.
```

Comment:
A few of these docs comments are missing the first header line (`##`).

Reply:
Added.